### PR TITLE
add dongchan as blog author

### PR DESCRIPTION
**Pull request by gywndi**
Hi, 
As we discussed with Daniil, we upload the author content before uploading the content.
Thanks.
Chan.

### CODE REVIEW

**Review by dbazhenov — COMMENTED** [review]
Great, thank you very much.

**Review by dbazhenov — COMMENTED** [review]
ready

File: content/blog/2021-07-21-create-your-own-exporter-in-go.md [hunk 1]
```diff
@@ -0,0 +1,747 @@
+---
+title: 'Create your own Exporter in Go!'
+date: Wed, 21 Jul 2021 09:42:53 +0000
+draft: false
+tags: ['Exporter', 'Go', 'Query', 'MySQL', 'Prometheus' 'Programming']
```
Comment (dbazhenov) [inline content/blog/2021-07-21-create-your-own-exporter-in-go.md]: Missing comma between Prometheus and Programming

**Comment by gywndi** [comment]
Hi, 

I think I miss “,” in my post.
Fix and push again.

Thanks.
Chan.

**Review by dbazhenov — CHANGES_REQUESTED** [review]
A missing comma between Prometheus and Programming in the post tags

File: content/blog/2021-07-21-create-your-own-exporter-in-go.md (hunk 1 above)
Comment (gywndi) [reply content/blog/2021-07-21-create-your-own-exporter-in-go.md]: Ok, I add comma.

**Review by dbazhenov — APPROVED** [review]

**Comment by dbazhenov** [comment]
@gywndi 
Checked locally and made minor changes to headers and date.
https://github.com/percona/community/pull/50

**Comment by dbazhenov** [comment]
Congratulations, your post is published, thank you very much!
https://percona.community/blog/2021/07/21/create-your-own-exporter-in-go/

**Comment by gywndi** [comment]
Hi,

While reading the blog post, I noticed that the article was duplicated.
It's not that the content is wrong, so it's not urgent, but please apply it.

Thanks.
Chan.

**Comment by dbazhenov** [comment]
@gywndi 
I think you're talking about the "Did you like this post? Why not read more?" recommendation block at the end of the article. This is an error in the block, I will try to fix it.

**Pull request merged**